.yarn/patches/debug-npm-4.3.4-4513954577.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/src/browser.js b/src/browser.js
+index cd0fc35d1ee11e0d6e15421021a54c18958e04d9..c0cfdf294fb0428c4d7f4c41ad641c8027e15197 100644
+--- a/src/browser.js
++++ b/src/browser.js
+@@ -11,7 +11,9 @@ exports.useColors = useColors;
+ exports.storage = localstorage();
+ exports.destroy = (() => {
+ 	let warned = false;
+-
++	if(!warned){
```

Comment:
Did you test this ?

Reply:
<img width="741" alt="image" src="https://user-images.githubusercontent.com/525211/170010799-71a6b039-2d68-4105-bfd5-5d18669638de.png">

Looks like it works, though I don't really know how